lib/interfaces/manual.js

```diff
@@ -26,68 +26,7 @@ var manualRequestInformationExtractor =
     require('../request-extractors/manual.js');
 var errorHandlerRouter = require('../error-router.js');
 
-/**
- * The interface for manually reporting errors to the Google Error API in
- * application code, this interface takes five arguments, three of which can
- * be supplied by application-code invocation and only one of which is required.
- * The first arugment `client` will be bound to an instance of this function so
- * that this handler can send error information back to Error Reporting API. The
- * second argument `err` is the actual error information that the application is
- * attempting to report. This data can be of any type, but better errors will be
- * logged given a valid instance of Error/Exception class, this is the only
- * required argument to be supplied during invocation. The third argument
```

Comment:
This description talks in detail about the number of arguments, which ones are optional, and what their purpose is. I have moved such details to the @-param documentation where it made sense.